CHANGELOG.md

```diff
@@ -4,7 +4,23 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## [unreleased] - XXXX-XX-XX
+## [Unreleased] 
+
+- 
+
+
+## [1.1.0] - 2022-07-26
```

Comment:
Dates like this end up being quite a long way different from the date of the release tag and release tarball.
But not really important.